### DISCUSSION TITLE
How do I update database schema version to latest?

### DISCUSSION THREAD

**Discussion by alephalpha0** [stashapp/stash#3447] [Q&A] [answered]
After downloading #3436 and killing the v0.18.0 process and switching out the binaries and restarting stash.. the startup logs had this warning in them:

```
INFO[2023-02-16 22:42:16] using config file: /home/dave/.stash/config.yml
DEBU[2023-02-16 22:42:16] Reading plugin configs from /media/dave/coldStorage/.stash/plugins
DEBU[2023-02-16 22:42:16] Reading scraper configs from /media/dave/coldStorage/.stash/scrapers
WARN[2023-02-16 22:42:16] database schema version 41 does not match required schema version 43
DEBU[2023-02-16 22:42:16] Loading custom performer images from /media/dave/coldStorage/.stash/custperf
INFO[2023-02-16 22:42:16] stash version: v0.19.0 (8bd5f91e) - Official Build - 2023-02-13 23:34:29
INFO[2023-02-16 22:42:16] stash is listening on 0.0.0.0:9999
INFO[2023-02-16 22:42:16] stash is running at http://localhost:9999/
INFO[2023-02-16 22:42:17] Version v0.19.0 (8bd5f91e) is already the latest released
```

I'm completely at a loss as how to update the db's schema to latest.. or really even where that information would come from..

**Comment by WithoutPants** [comment, marked as answer]
Just load up the stash webpage as you normally would. You will be prompted to migrate your database.

**Comment by alephalpha0** [comment]
:face_exhaling: :man_facepalming:
One lets stash update its db scheme to the latest version by accessing stash through a ***fresh*** browser tab or window.. 

I was using the old tab used to access v0.18.0.. didn't even reload the cache.
